app/views/admin/users/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= form_for @user, :url => {:controller => 'admin/users', :action => 'update'} do |f| %>
```

Comment:
Tried to use a partial here but wasn't working. I had to specify the controller because otherwise, it would go to the regular users controller's update action

Reply:
Good workaround

Reply:
We could try putting the forms content in a partial and only having the form for in the main document when the time comes that we want to update this, but this definitely works for now, I wouldn’t have thought of overriding the controller action!